### PR TITLE
Update supported platforms

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

**Comment by antiagainst** [comment]
Here is what it looks like: https://github.com/antiagainst/shaderc/blob/platforms/DEVELOPMENT.howto.md

File: DEVELOPMENT.howto.md [hunk 1]
```diff
@@ -43,7 +43,12 @@ We expect Shaderc to always build and test successfully on the platforms listed
 below.  Please keep that in mind when offering contributions.  This list will
 likely grow over time.
 
-* Linux x86
+| Platform | Build Status |
+|:--------:|:------------:|
+| Android (ARMv7)  | |
```
Comment (dmircevski) [inline DEVELOPMENT.howto.md]: Should we leave the Android status mysteriously blank, say something like "not automated," or list the status of the most recent manual build?

Comment (antiagainst) [inline DEVELOPMENT.howto.md]: Reasonable. Filled the blank with "Not Automated". :)

**Comment by dmircevski** [comment]
+2

File: DEVELOPMENT.howto.md [hunk 2]
```diff
@@ -43,7 +43,12 @@ We expect Shaderc to always build and test successfully on the platforms listed
 below.  Please keep that in mind when offering contributions.  This list will
 likely grow over time.
 
-* Linux x86
+| Platform | Build Status |
+|:--------:|:------------:|
+| Android (ARMv7)  | Not Automated |
+| Linux (x86_64)   | [![Linux Build Status](https://travis-ci.org/google/shaderc.svg)](https://travis-ci.org/google/shaderc "Linux Build Status") |
+| Windows (x86_64) | [![Windows Build status](https://ci.appveyor.com/api/projects/status/xcxci2mekggib1an?svg=true)](https://ci.appveyor.com/project/antiagainst/shaderc "Windows Build Status") |
```
Comment (dneto0) [inline DEVELOPMENT.howto.md]: Might be nice to say what compiler is supported.  It's especially important for Windows.

**Comment by dneto0** [comment]
+2

File: DEVELOPMENT.howto.md (hunk 2 above)
Comment (antiagainst) [inline DEVELOPMENT.howto.md]: @AWoloszyn is an expert on that. I'll let him handle it. :)

Comment (dneto0) [inline DEVELOPMENT.howto.md]: +2

**Pull request merged**